feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -289,87 +289,66 @@ func pushDefaultEntries(t *testing.T, args *testArgs, nextHops []string) []strin
 	return virtualVIPs
 }
 
-// createVrf creates takes in a list of VRF names and creates them on the target devices.
-func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string) {
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+	d := &oc.Root{}
+
+	vrfs := []string{*deviations.DefaultNetworkInstance, vrf1, vrf2, vrf3}
+	createVrf(t, dut, vrfs)
+
+	// configure Ethernet interfaces first
+	configureInterfaceDUT(t, d, dut, dp1, "src")
+	configureInterfaceDUT(t, d, dut, dp2, "dst")
+
+	// configure an L3 subinterface without vlan tagging under DUT port#1 and assign it to vrf1
+	createSubifDUT(t, d, dut, dp1, 0, 0)
+	fptest.AssignToNetworkInstance(t, dut, dp1.Name(), vrf1, 0)
```

Comment:
also please move fptest.AssignToNetworkInstance(t, dut, dp1.Name(), vrf1, 0) to the end of function. note that there is a deviation InterfaceConfigVrfBeforeAddress that can be used to call fptest.AssignToNetworkInstance either at the beginning or end of the function.

Reply:
Same comment here as I left in the other pr:

Could you elaborate on this?
Previously I've been told that IP addresses have to be configured after interfaces are assigned to VRFs in some implementations.. Is that not the case anymore?

Reply:
I think it is still the case for other vendors.  that is why it safe to use InterfaceConfigVrfBeforeAddress.